lib/net/networking_engine.js

```diff
@@ -610,10 +588,15 @@ shaka.net.NetworkingEngine = class extends shaka.util.FakeEventTarget {
       };
       request.requestStartTime = Date.now();
 
+      let minBytesForProgressEvents = 16e3;
+      if (this.config_) {
```

Comment:
Instead of checking if this.config_ is non-null, could we require configure() to be called before use?  Then we don't have to duplicate default values here.

Reply:
Done!